### ISSUE TITLE
`asia` dataset is missing in examples folder

### ISSUE THREAD

**Issue by GiggleLiu** [mroavi/JunctionTrees.jl#6]
I am reading this documentation page, but I can not reproduce it without model data
https://mroavi.github.io/JunctionTrees.jl/stable/usage/

I would be great if the `asia` dataset can be in the examples folder.

**Comment by mroavi** [comment]
The ASIA dataset is located inside the docs dir: https://github.com/mroavi/JunctionTrees.jl/tree/master/docs/src/problems/asia

**Comment by mroavi** [comment]
Yeah, maybe it would be a good idea to use only one dir for all problem datasets.

**Comment by GiggleLiu** [comment]
👍🏾 , Putting them together sounds good. Maybe also mention the path in this documentation page too.
